src/main/java/com/powsybl/openloadflow/DefaultOuterLoopConfig.java

```diff
@@ -43,6 +43,17 @@ private static OuterLoop createShuntVoltageControlOuterLoop(OpenLoadFlowParamete
         }
     }
 
+    private static OuterLoop createPhaseShifterControlOuterLoop(OpenLoadFlowParameters parametersExt) {
+        switch (parametersExt.getPhaseShifterControlMode()) {
+            case CONTINUOUS_WITH_DISCRETISATION:
```

Comment:
`CONTINUOUS_WITH_DISCRETISATION` is a good parameter name, but it is not really coherent with the choices we have made for generator and shunt voltage control. For both we have the incremental mode but also a `WITH_GENERATOR` mode. So what is your proposal to make everything more coherent?